manageiq-schema.gemspec

```diff
@@ -19,13 +19,13 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "ancestry"
-  spec.add_dependency "activerecord-id_regions", "~> 0.3.0"
+  spec.add_dependency "activerecord-id_regions", "~> 0.3.0", ">= 0.3.1"
```

Comment:
```suggestion
  spec.add_dependency "activerecord-id_regions", "~> 0.3.1"
```

Reply:
@bdunne I already changed this prior to even seeing this, but this isn't a pattern that I pulled out of thin air, as it is something we commonly do:

https://github.com/ManageIQ/manageiq/blob/a94268f6aab6d41b691fdb0ea8c07712f357aa61/Gemfile#L36

I didn't want to leave it to chance that it would work though, so I tried a slightly more restrictive approach by "raising the floor".  However, it seems like `bundler` wasn't having it:

https://travis-ci.com/github/ManageIQ/manageiq-cross_repo-tests/builds/207873090

So I reverted back.  Though, I don't think it was wrong in at least considering it.

Reply:
I think it makes sense to specify `"~> x.y", ">= x.y.1"` when we want our dependency to be `~> x.y` (letting y & z float), but require at least `x.y.1` because there was a security bug or something in `x.y.0`.  But in a case like this where we're already locked down to z, I think it adds extra complexity.

i.e. If activerecord-id_regions was 1.0, I agree that we would probably want `"~> 1.0", ">= 1.0.1"`

Reply:
But since it doesn't fix a security issue, we probably don't need to change anything since bundler should resolve properly as is.

Reply:
In this case was more being explicit, and not necessarily requiring it.  I think this was more residual habit from days of 1.4 bundler where `molinillo` would be pretty good, but have a few edge cases where it couldn't properly resolve things and got confused.

But because of the other bump in the `rails` dependency in this gem, `manageiq` will always resolve to `< 0.3.1` for Rails 5.x and `>= 0.3.1` for Rails 6.0ish, so you are correct and going with just the `~> 0.3.0` we had is fine (again, what I already changed it back to).